flow-minor-mode.el

```diff
@@ -214,6 +231,7 @@ BODY progn"
       (goto-char (point-min))
       (forward-line 1)
       (delete-region (point) (point-max))
+      (flow-minor-colorize-buffer)
```

Comment:
Thanks for the update, looks good (but i have zero time test it myself), also why do we need this line here?

Reply:
It sets up font-lock for the eldoc message in the `*Flow Eldoc*` buffer.  Is there a better way?

Reply:
Ok, got it.

Reply:
Thanks for the PR again.